### ISSUE TITLE
Allow gitmojis

### ISSUE THREAD

**Issue by rubensa** [conventionalcommit/commitlint#15]
Allow using [gitmojis](https://gitmoji.dev/) in the commit message `description`.

Looks like they are detected as part of the `scope`.

For example, the following commit message generated in **VSCode** using [Conventional Commits extension](https://marketplace.visualstudio.com/items?itemName=vivaxy.vscode-conventional-commits):
```
chore: :hammer: Add commitlint as git commit-msg hook

Also add **Conventional Commits** VSCode extension
```

gives:
```bash
commitlint

→ input: "chore: :hammer: Add co..."

Errors:
  ❌ parser: scope must be followed by ': '

Total 1 errors, 0 warnings, 0 other severities
```

**Comment by gumbo2k** [comment]
Found it! The bug is not in `commitlint` itself, but in [conventionalcommit/parser](https://github.com/conventionalcommit/parser)

And it is slightly different than I thought.
The parser doesn't work with regex but uses a lexer instead.

When the parser reads a header like `type(scope): description` and moves from `type(scope)` to `description` it uses a _greedy_  `take(": ")` to read the delimiter.
This `take` will read any number (and order) of  ` ` and `:` into the current token. and then the parser compares the result to `: ` . 
This fails when the `description` starts with a `:` as the `take` will have read `: :` already.
I've fixed it in my fork and I'll open a PR.
